utils.ts

```diff
@@ -94,11 +172,16 @@ const filterOutDuplicateStops = (
   // added to the addendum field in Pelias. Therefore, there is still potential
   // for some transit stops without the "operator" tag to still be included in
   // search results.
+  // In HERE, transit stops will have one of the following categories.
+  const HERE_TRANSIT_STOP_CATEGORIES = ['Underground Train-Subway', 'Bus Stop']
```

Comment:
What about something like a ferry terminal or cable car base station or something?

Reply:
I found a better way to detect these categories! All transit categories begin with an ID of 400, so a simple regex like this should work.